src/app.js

```diff
@@ -1,10 +1,21 @@
 import { quizData } from './data.js';
 import { initWelcomePage } from './pages/welcomePage.js';
+import { initQuestionPage } from './pages/questionPage.js';
 
 const loadApp = () => {
-  quizData.currentQuestionIndex = 0;
+  // quizData.currentQuestionIndex = 0;
+  if (localStorage.questions === undefined || JSON.parse(localStorage.questions).length >= quizData.questions.length) {
```

Comment:
This doesn't look like correct use of `localStorage`. Checkout the [docs](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage).

You are supposed to use `setItem` and `getItem`.